app/initializers/ember-cli-mirage.js

```diff
@@ -11,7 +17,7 @@ export default {
 
     if (_shouldUseMirage(env, ENV['ember-cli-mirage'])) {
       var modulesMap = readModules(ENV.modulePrefix);
-      var hasFactories = !Ember.isEmpty(modulesMap['factories']);
+      var hasFactories = hasFactories(modulesMap);
```

Comment:
build error was

> 'undefined' is not a function (evaluating 'hasFactories(modulesMap)')

I wonder if it's because `var hasFactories` here overrides the function declaration above?